### ISSUE TITLE
Disconnecting too frequently.

### ISSUE THREAD

**Issue by sksar** [localtunnel/localtunnel#167]
The service is disconnecting 3-4 minutes later i set up a tunnel. This is happening everytime and i am not getting a constant connection. Although my server is up and running but the connections cut too soon.

**Comment by kirillweb** [comment]
try to use [localtunnel-restarter](https://github.com/kirillweb/localtunnel-restarter)

**Comment by AravinthChandran** [comment]
Guys,
I'm using localtunnel on a daily basis, this frequent disconnection is giving me a hard time to debug webhooks . How much work is needed to fix this ? If some one can give directions i can work on this.

**Comment by k9t9** [comment]
Maybe related or duplicate https://github.com/localtunnel/localtunnel/issues/177

**Comment by sksar** [comment]
no @k9t9, nothing related to #177 .... 
I think the problem is arising due to slow network or heavy ping delay....

**Comment by Quacky2200** [comment]
I have to use localtunnel-restarter too however I think I have found a temporary fix.

I'm also not sure if this will always be related to you but I found less disconnections when commenting out a couple of lines in ` lib/Tunnel.js`. Lines 47-50 'error out' when a response is not a 200 which is pretty idiotic personally... (as it would mean that a missing image would break the connection and would be another iteration for localtunnel-restarter. I haven't seen very many disconnections since.

https://github.com/localtunnel/localtunnel/blob/578dc9aaae433c118781482f6bce749f2c554967/lib/Tunnel.js#L45-L51
